debian/leap-mx.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Leap MX
+After=network.target
```

Comment:
I think leap-mx doesn't need network up to work, but it should be turn on before postfix. I don't know much about systemd, is there a `Before=postfix.target` or something? or doing it after network is the best we can do.